NextGenConnect/JSON to HL7v2.xml

```diff
@@ -143,6 +143,8 @@ else{
 
 //PID-10 Race  -- MVS-528
 
+tmp[&apos;PID&apos;][&apos;PID.10&apos;][&apos;PID.10.1&apos;] = patient[&apos;extension&apos;][0][&apos;extension&apos;][0][&apos;valueCoding&apos;][&apos;code&apos;];
+tmp[&apos;PID&apos;][&apos;PID.10&apos;][&apos;PID.10.3&apos;] = &apos;HL70005&apos;;
```

Comment:
Can you put the display value in the second component?

Reply:
No. It's required to be in the 3rd component.

Reply:
No, I mean the display value from extension.extension.valueCoding.display. It would be additional information, not changing what you have in the third component now.